lib/pages/scout/DesktopScout.dart

```diff
@@ -139,13 +142,46 @@ class _DesktopScoutState extends State<DesktopScout> {
             setState(() {
               sportState = 1;
             });
+          bloc.add(SelectSport(selectedSport: SupportedSport.MLB));
         },
         child: Text("MLB",
             style: textSwapState(
                 sportState == 1,
                 textStyle(Colors.white, sportFilterTxSz, false, false),
                 textStyle(Colors.amber[400]!, sportFilterTxSz, false, true))),
       )),
+      Container(
+          child: TextButton(
+        onPressed: () {
+          myController.clear();
+          if (sportState != 2)
+            setState(() {
+              sportState = 2;
+            });
+          bloc.add(SelectSport(selectedSport: SupportedSport.NFL));
```

Comment:
We don't want to add NFL and NBA back to UI

Reply:
Done. I removed them from the Scouts Page